c/iotdm_client/src/iotdm_client.c

```diff
@@ -487,7 +490,14 @@ unsigned long get_milliseconds()
     xtime_get(&tm, TIME_UTC);
 
     return (tm.sec + (tm.nsec / 1000000L));
-
+#elif __MACH__
```

Comment:
We need to use our platform abstraction for getting the tick count, rather than the code in this function.  If we do that work, this change can go away...